core/src/test/scala/sttp/client3/testing/HttpTest.scala

```diff
@@ -53,6 +54,7 @@ trait HttpTest[F[_]]
   protected def throwsExceptionOnUnsupportedEncoding = true
   protected def supportsHostHeaderOverride = true
   protected def supportsCancellation = true
+  protected def supportsAutoDecompressionDisabling = false
```

Comment:
sorry, one more comment. I think it's better to require by default that a backend passes all tests (supports all features), and only selectively mark the backends that don't support something. So I'd set this as `true` by default, and override with `false` if needed